app.js

```diff
@@ -24,11 +24,15 @@ const capitalized = string => string[0].toUpperCase() + string.slice(1).toLowerC
 
 app.locals.title = `${capitalized(projectName)}- Generated with Ironlauncher`;
 
-// 👇 Start handling routes here
+// Start handling routes here
 const index = require('./routes/index');
 app.use('/', index);
 
-// ❗ To handle errors. Routes that don't exist or errors that you handle in specific routes
+const celebrities = require('./routes/celebrities.routes')
+app.use('/celebrities',celebrities)
+
+app.use("/movies",require('./routes/movies.route'))
```

Comment:
Eu mudaria o nome do seu arquivos pra `.routes` pra manter a consistência com o nome do outro arquivo

```suggestion
app.use("/movies", require('./routes/movies.routes'))
```